src/main/java/core/basesyntax/service/performer/FruitTransactionPerformerImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.performer;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.operation.OperationStrategy;
+
+public class FruitTransactionPerformerImpl implements Performer {
+    private final OperationStrategy operationStrategy;
+    private final FruitTransactionDao fruitTransactionDao;
+
+    public FruitTransactionPerformerImpl(OperationStrategy operationStrategy,
+                                         FruitTransactionDao fruitTransactionDao) {
+        this.operationStrategy = operationStrategy;
+        this.fruitTransactionDao = fruitTransactionDao;
+    }
+
+    @Override
+    public void performAll() {
```

Comment:
you can pass to this method all `transactions` without using of `fruitTransactionDao`